backend/api/v2beta1/recurring_run.proto

```diff
@@ -85,9 +86,11 @@ message RecurringRun {
   // recurring run. Can be either a pipeline version id, or a pipeline spec.
   oneof pipeline_source {
     // The ID of the pipeline version used for creating runs.
-    string pipeline_version_id = 4;
+    string pipeline_version_id = 4 [deprecated=true];
     // The pipeline spec.
     google.protobuf.Struct pipeline_spec = 5;
+    // Reference to a pipeline version containing pipeline_id and pipeline_version_id.
+    PipelineVersionReference pipeline_version_reference = 18;
```

Comment:
6?

Reply:
NVM

Reply:
`6` is used by `runtime_config`. The next number is `18`. Similarly, I used `18` in `run.proto`.